src/components/CityWeatherShortcut/CityWeatherShortcut.tsx

```diff
@@ -21,14 +22,16 @@ const CityWeatherShortcut = ({ cityName }: Props) => {
     const roundedTemperature = Math.round(temp_c);
 
     return (
-      <Container data-testid={CityWeatherShortcutTestIds.Widget}>
-        <Icon src={condition.icon} alt="weather image" />
-        <Temperature>
-          <Value>{roundedTemperature}</Value>
-          <TemperatureSign>&deg;C</TemperatureSign>
-        </Temperature>
-        <City>{data.location.name}</City>
-      </Container>
+      <Tooltip title={data.location.name}>
```

Comment:
please use destructing for `name` and reuse it as a name of city

Reply:
@bednarskimichal done